### PR TITLE
Fix issue in loading the node flag image.

### PR DESCRIPTION
**Pull request by Mahmoud-Emad**
### Description

- The issue was identified as a result of errors in the image link, leading to problems with image reloading.
- The necessary changes were made to the code, particularly in the image loading section, to rectify the issue.

### Changes

- Resolved the issue related to image reloading, ensuring that the bug no longer occurs.
- Implemented a fix to address the error encountered while attempting to load certain images.
- Replaced the hard-coded flag source with a dynamic function call (`getCountryFlagSrc`) for improved flexibility and accuracy.
- Added a tooltip displaying the node's country information.

### Related Issues

- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/2024

### Screenshots

- image
- image
- image

### Checklist

- [ ] Tests included
- [x] Build pass
- [x] Documentation
- [x] Code format and docstrings
- [x] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

**Comment by AlaaElattar** [comment]
- This one still not loading

image

File: packages/playground/src/components/node_selector/TfNodeDetailsCard.vue [hunk 1]
```diff
@@ -45,6 +50,13 @@
           </VChip>
         </template>
       </VTooltip>
+      <VTooltip text="Node Country" v-if="node && node.location.country" location="left">
```
Comment (AlaaElattar) [inline packages/playground/src/components/node_selector/TfNodeDetailsCard.vue:53]: The county name is already displayed in a chip, why to show it again in the tooltip ?

**Comment by AlaaElattar** [comment]
image


> * This one still not loading
> 
> image

**Comment by samaradel** [comment]
```Switzerland``` flag not loaded

image

File: packages/playground/src/components/node_selector/TfNodeDetailsCard.vue (hunk 1 above)
Comment (Mahmoud-Emad) [reply packages/playground/src/components/node_selector/TfNodeDetailsCard.vue:53]: @ehab-hassan requested the design. Would you like to remove it?

**Comment by Mahmoud-Emad** [comment]
> `Switzerland` flag not loaded
> 
> image

it worked with me 

- image
- image

**Review by AlaaElattar — APPROVED** [review]
Great Job

**Review by samaradel — APPROVED** [review]
good job!

**Pull request merged**